.vib/grafana-loki/cypress/cypress/integration/grafana_loki_spec.js

```diff
@@ -51,11 +52,13 @@ it('checks Loki range endpoint', () => {
 });
 
 it('can publish and retrieve a label', () => {
+  // Per configuration, Loki rejects timestamps older than 168h
+  const updatedJson = JSON.stringify(body).replace('timestamp_placeholder', lastMinuteTimestamp)
   cy.request({
     method: 'POST',
     headers: { 'Content-Type': 'application/json; charset=utf-8' },
     url: 'loki/api/v1/push',
-    body: body,
+    body: JSON.parse(updatedJson),
```

Comment:
Small remark: I'd name this `upToDateJson` instead of `updatedJson`, just to make a clear reference on the time it was generated. WDYT?